src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -531,4 +533,226 @@ private double convertInch1200ToPixels(double inch1200Value){
         final int DPI = 300;
         return inch1200Value * DPI / 1200.0;
     }
+
+
+    /************************************* Streaming implementation **********************************************/
+
+    /**
+     * Get images of newspaper pages with given query present in text.
+     * @param query     to search for.
+     * @param startTime is the earliest boundary for the query.
+     * @param endTime   is the latest boundary for the query.
```

Comment:
Please state if the boundaries are inclusive or exclusive. I am pretty sure they are both inclusive here, but Java methods often use inclusive for the starting point and exclusive for the ending point so it is customary to be explicit about it.

Reply:
…and maybe call them `startYear` and `endYear`? "Time" is a bit abstract.

Reply:
Yes!